src/components/explorer/GuildCard.tsx

```diff
@@ -33,23 +38,38 @@ const GuildCard = ({ guildData }: Props): JSX.Element => (
   >
     <DisplayCard>
       <SimpleGrid
-        templateColumns={image ? "3rem calc(100% - 4.25rem)" : "1fr"}
+        templateColumns={image ? "3rem calc(100% - 5.25rem)" : "1fr"}
         gap={4}
         alignItems="center"
       >
         {image && <GuildLogo imageUrl={guildData.imageUrl} />}
         <VStack spacing={2} alignItems="start" w="full" maxW="full" mb="0.5" mt="-1">
-          <Text
-            as="span"
-            fontFamily="display"
-            fontSize="lg"
-            fontWeight="bold"
-            letterSpacing="wide"
-            maxW="full"
-            noOfLines={1}
-          >
-            {guildData.name}
-          </Text>
+          <Box position="relative">
+            <Text
+              as="span"
+              fontFamily="display"
+              fontSize="lg"
+              fontWeight="bold"
+              letterSpacing="wide"
+              maxW="full"
+              noOfLines={1}
+            >
+              {guildData.name}
+              {guildData.tags?.includes("VERIFIED") && (
+                <Tooltip label="Verified" hasArrow>
+                  <Circle
```

Comment:
The icon color is wrong in light mode. Also, should make a VerifiedIcon component and use that here and in the guild page for the title, so the design stays consistent